modules/violation-view-policy-tab.adoc

```diff
@@ -9,21 +9,34 @@
 The *Policy* tab of the *Details* panel displays details of the policy that caused the violation.
 
 [discrete]
-== Policy Details section
+== Policy overview section
 
-The policy details section lists the following information:
+The policy overview section lists the following information:
 
-* *Id*: The numerical identifier for the policy.
-* *Name*: The name of the policy.
+* *Severity* - The risk level for the violation.
+* *Categories*: The policy category of the policy.
+* *Type*:
 * *Description*: A detailed explanation of what the policy alert is about.
 * *Rationale*: Information about the reasoning behind the establishment of the policy and why it matters.
-* *Remediation*: Suggestions on how to fix the violation.
-* *Enabled*: Indicates if the policy is enabled.
-* *Categories*: The policy category of the policy.
+* *Guidance*: Suggestions on how to address the violation.
+* *MITRE ATT&CK*: Indicates whether or not the policy has MITRE ATT&CK vectors.
+
+//These no longer seem to be in the GUI:
+//* *Id*: The numerical identifier for the policy.
+//* *Name*: The name of the policy.
+//* *Enabled*: Indicates if the policy is enabled.
```

Comment:
A policy is enabled (or was enabled) if a violation was generated so this is redundant.